igor.dzetcyna/first/task 1/array.rb

```diff
@@ -0,0 +1,8 @@
+# Заповнити випадковими числами від -10 до 10 двовимірний масив 3х3 елементів
+arr = Array.new(3) { Array.new(3) { rand -10..10 } }
+p arr
+# Перетворимо багатовимірний масив в одновимірний і знаходимо все позитивні числа
+arr1 = arr.flatten.find_all { |el| el > 0 }
+puts arr1
+print "Summa = "
```

Comment:
You could interpolate arr1.sum to this string, but if there is no interpolation prefer single quoted strings.
https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals-single-quote